app/controllers/expenditures_controller.rb

```diff
@@ -0,0 +1,72 @@
+class ExpendituresController < ApplicationController
+  before_action :set_expenditure, only: %i[show edit update destroy]
+  before_action :authenticate_user!
+
+  # GET /expenditures or /expenditures.json
+  def index
+    @expenditures = Expenditure.all.where(user: current_user).order('created_at DESC')
+  end
+
+  # GET /expenditures/1 or /expenditures/1.json
+  def show; end
+
+  # GET /expenditures/new
+  def new
+    @expenditure = Expenditure.new
+    @categories = Category.all.where(user: current_user)
+  end
+
+  # GET /expenditures/1/edit
+  def edit; end
+
+  # POST /expenditures or /expenditures.json
+  def create
+    @expenditure = Expenditure.new(expenditure_params)
+    @expenditure.author_id = current_user.id
+
+    respond_to do |format|
+      if @expenditure.save
+        format.html { redirect_to categories_path, notice: 'Expenditure was successfully created.' }
+        format.json { render :show, status: :created, location: @expenditure }
+      else
+        format.html { redirect_to expenditures_path, notice: 'Select a category to create a new expense' }
+        format.json { render json: @expenditure.errors, status: :unprocessable_entity }
+      end
+    end
+  end
```

Comment:
- [ ] when creating an expense it should redirect you to the transaction page now it takes me to categories page please make sure to do that in order to have a better UX and also match the requirements 
- Screenshot of the requirements 
<img width="1016" alt="Screen Shot 2022-10-14 at 18 28 20" src="https://user-images.githubusercontent.com/61790587/195906697-bce33421-4eee-4866-973e-f87632042521.png">